src/foundry/client/pixi/webgl/shaders/filters/visibility.d.mts

```diff
@@ -0,0 +1,57 @@
+export {};
+
+declare global {
+  /**
+   * Apply visibility coloration according to the baseLine color.
+   * Uses very lightweight gaussian vertical and horizontal blur filter passes.
+   */
+  class VisibilityFilter extends AbstractBaseMaskFilter {
+    constructor(...args: ConstructorParameters<typeof AbstractBaseMaskFilter>);
+
+    /**
+     * @defaultValue
+     * ```js
+     * {
+     *    exploredColor: [1, 1, 1],
+     *    unexploredColor: [0, 0, 0],
+     *    screenDimensions: [1, 1],
+     *    visionTexture: null,
+     *    primaryTexture: null,
+     *    overlayTexture: null,
+     *    overlayMatrix: new PIXI.Matrix(),
+     *    hasOverlayTexture: false
+     * }
+     * ```
+     */
+    static override defaultUniforms: AbstractBaseShader.Uniforms;
+
+    static override create(
+      uniforms?: AbstractBaseShader.Uniforms,
+      options?: Parameters<(typeof VisibilityFilter)["fragmentShader"]>,
+    ): VisibilityFilter;
+
+    static override vertexShader: string;
+
+    static override fragmentShader(options: { persistentVision: boolean }): string;
+
+    /**
+     * Set the blur strength
+     * @param value - blur strength
+     */
+    set blur(value: number);
+
+    get blur();
```

Comment:
You probably want `get blur(): number;` I'm not 100% sure why this isn't being caught by some option like `noImplicitAny`.

Reply:
Me neither! Fixed.